docs/changelog.txt

```diff
@@ -62,6 +62,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 
 ## Lua
 - Lua wrappers for functions reverse-engineered from ambushing unit code: ``isHidden(unit)``, ``isFortControlled(unit)``, ``getOuterContainerRef(unit)``, ``getOuterContainerRef(item)``
+- Added ``customRawData`` utility to Lua library.
```

Comment:
`customRawData` should be surrounded in single quotes, not double, to link to the documentation. also maybe add a note about what the module does.